### PR TITLE
AArch64: Fill _relocationTargetTypeToHeaderSizeMap

### PR DESCRIPTION
**Pull request by knn-k**
This commit fills the table _relocationTargetTypeToHeaderSizeMap for
AArch64.

Signed-off-by: KONNO Kazuhiro <konno@jp.ibm.com>

### CODE REVIEW

**Comment by knn-k** [comment]
This commit depends on #7222.

**Comment by knn-k** [comment]
#7222 is merged.  Removed "WIP:".

**Comment by 0xdaryl** [comment]
@dsouzai , would you mind reviewing this one as well please?

File: runtime/compiler/aarch64/codegen/J9AheadOfTimeCompile.cpp [hunk 1]
```diff
@@ -42,107 +42,106 @@ uint8_t *J9::ARM64::AheadOfTimeCompile::initializeAOTRelocationHeader(TR::Iterat
...
+   64,                                       // TR_ProfiledInlinedMethodRelocation     = 46
+   40,                                       // TR_MethodPointer                       = 47
+   32,                                       // TR_ClassPointer                        = 48
+   16,                                       // TR_CheckMethodExit                     = 49
+   24,                                       // TR_ValidateArbitraryClass              = 50
+   0,                                        // TR_EmitClass (not used)                = 51
+   32,                                       // TR_JNISpecialTargetAddress             = 52
+   32,                                       // TR_VirtualRamMethodConst               = 53
+   48,                                       // TR_InlinedInterfaceMethod              = 54
+   48,                                       // TR_InlinedVirtualMethod                = 55
```
Comment (dsouzai) [inline runtime/compiler/aarch64/codegen/J9AheadOfTimeCompile.cpp]: These should be 40; on all the other 64-bit platforms it is 40. Additionally, in what will eventually be the one and only table of sizes (work in progress), the size of the binary template for also ends up evaluating to 40 (if you follow the inheritance chain):
https://github.com/eclipse/openj9/blob/0739a278d3a5cbc0c19e14e7fcb4d8164379b9ea/runtime/compiler/runtime/RelocationRecord.cpp#L4668-L4669

Comment (knn-k) [reply runtime/compiler/aarch64/codegen/J9AheadOfTimeCompile.cpp]: It seems I overlooked the `if` below, and over-counted the sizes for those two types.
https://github.com/eclipse/openj9/blob/d01bf6fab7100415c9c78341cfefd95848610888/runtime/compiler/aarch64/codegen/J9AheadOfTimeCompile.cpp#L417-L418

**Comment by knn-k** [comment]
Updated the file reflecting the review comment.

**Pull request merged**